programs/witness_node/main.cpp

```diff
@@ -298,6 +298,14 @@ void write_default_logging_config_to_stream(std::ostream& out)
    out << "# declare an appender named \"stderr\" that writes messages to the console\n"
           "[log.console_appender.stderr]\n"
           "stream=std_error\n\n"
+          "# declare an appender named \"default\" that writes messages to default.log\n"
+          "[log.file_appender.default]\n"
+          "# filename can be absolute or relative to this config file\n"
+          "filename=logs/default/default.log\n"
```

Comment:
Perhaps better call it "node.log" or "chain.log"?

Reply:
I named it `default.log` since it's written by the `default` logger in `[logger.default]` section.